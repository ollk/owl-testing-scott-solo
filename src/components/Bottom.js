import React from 'react';
import './Bottom.css';

class Bottom extends React.Component {
  render() {
    return;
  }
}

export default Bottom;
