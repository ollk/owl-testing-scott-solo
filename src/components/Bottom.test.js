import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Bottom from './Bottom';

describe('<Bottom />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Bottom />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  
  it('matches previous snapshot', () => {
    const tree = renderer.create(<Bottom />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
