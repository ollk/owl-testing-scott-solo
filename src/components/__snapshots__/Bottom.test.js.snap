// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Bottom /> matches previous snapshot 1`] = `
<footer>
  <img
    alt="reactions"
    src="Reactions.png"
  />
  <img
    alt="media buttons"
    src="mediaIcons.png"
  />
  <button
    className="joinButton"
  >
    Join Stage
  </button>
</footer>
`;
